Root/Krzysztof/Root/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link href="https://fonts.googleapis.com/css?family=Anton" rel="stylesheet">
+		<link type="text/css" rel="stylesheet" href="lightbox/dist/css/lightbox.css"/>
+		<link type="text/css" rel="stylesheet" href="stylesheet.css"/>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+		<title>Krzysztof Marcinkowski</title>
+	</head>
+	<body>
+		<div class="Header">
+			<h1>Krzysztof Marcinkowski</h1>
+		</div>
+		<div class="menu">
+			<a href="metal.html"><button id="metal">METAL WORK</button></a>
+			<a href="wood.html"><button id="wood">WOOD WORK</button></a>
+		</div>
+    <div class="home">
+      <a href="index.html"><button id="homebutton">HOME</button></a>
```

Comment:
Generally `<button>` should be used in forms. You can use it like you are for styling but generally the styling in most browsers is not that nice.